src/motion_input_shaping/shaped_axis.py

```diff
@@ -221,12 +221,18 @@ def move_relative(
                 min(self.get_setting_from_lockstep_axes("motion.decelonly", Units.NATIVE))
                 != deceleration_native
             ):
-                self.set_lockstep_axes_setting(
-                    "motion.decelonly", [deceleration_native], Units.NATIVE
-                )
+                if deceleration_native > 0:
+                    self.set_lockstep_axes_setting(
+                        "motion.decelonly", [deceleration_native], Units.NATIVE
+                    )
+                else:
+                    self.set_lockstep_axes_setting("motion.decelonly", [1], Units.NATIVE)
```

Comment:
You can potentially simplify this - instead of a large if-else block and repeating your call to the function, use `max(1, deceleration_native)` in a single statement.

Reply:
Done